Documentation/operations/performance/tuning.rst

```diff
@@ -12,6 +12,106 @@ Tuning Guide
 
 This guide helps you optimize a Cilium installation for optimal performance.
 
+Recommendation
+==============
+
+The default out of the box deployment of Cilium is focused on maximum compatibility
+rather than most optimal performance. If you are a performance-conscious user, here
+are the recommended settings for operating Cilium to get the best out of your setup.
+Each of the settings for the recommended performance profile are described in more
+detail on this page and in this `KubeCon talk <https://sched.co/1R2s5>`__:
+
+- netkit device mode
+- eBPF host-routing
+- BIG TCP for IPv4/IPv6
+- Bandwidth Manager (optional, for BBR congestion control)
+
+**Requirements:**
+
+* Kernel >= 6.8
+* Supported NICs for BIG TCP: mlx4, mlx5, ice
+
+To enable the first three settings:
+
+.. tabs::
+
+    .. group-tab:: Helm
+
+       .. parsed-literal::
+
+           helm install cilium |CHART_RELEASE| \\
+             --namespace kube-system \\
+             --set routingMode=native \\
+             --set bpf.datapathMode=netkit \\
+             --set bpf.masquerade=true \\
+             --set ipv6.enabled=true \\
+             --set enableIPv6BIGTCP=true \\
+             --set ipv4.enabled=true \\
+             --set enableIPv4BIGTCP=true \\
+             --set kubeProxyReplacement=true
```

Comment:
Nit: missing 

```
--set k8sServiceHost=${API_SERVER_IP} \
--set k8sServicePort=${API_SERVER_PORT}
```

https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/#quick-start

Reply:
Ok, I will add this as a follow-up given the older/existing sections need this too. Between netkit and netkit-l2 there should not be much diff in terms of performance, netkit in L3 mode basically tells the Pod that it doesn't need to resolve ARP which is great since it's not really necessary anyway. I can add a note.